code/modules/projectiles/guns/ballistic/pistol.dm

```diff
@@ -75,7 +75,7 @@
 	icon_state = "flatgun"
 
 /obj/item/gun/ballistic/automatic/pistol/stickman/pickup(mob/living/user)
-	..()
+	SHOULD_CALL_PARENT(FALSE)
```

Comment:
This needs a comment or should call the parent. We shouldn't be completely bypassing the protections in place without any reasoning at to why we are doing so

Reply:
This code is awful.

Fixed in b093cf0